python/lsst/daf/butler/script/removeRuns.py

```diff
@@ -0,0 +1,121 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from collections import defaultdict
+from dataclasses import dataclass
+from functools import partial
+from typing import cast, Callable, Dict, List, Mapping, Sequence, Tuple
+
+from .. import Butler
+from ..registry import CollectionType, MissingCollectionError
+from ..registry.queries import DatasetQueryResults
+
+
+@dataclass
+class RemoveRunsResult:
+    """Container to return to the cli command; contains the names of runs that
```

Comment:
Please put the content after the semi-colon on a new line with a blank line between so that the class summary is distinct from the explanation.